### PR TITLE
India/UDISE+ Optimise Process and Code

### PR DESCRIPTION
**Pull request by thejeshgn**
- Moved common variables to base class
- Use base folder when possible

### CODE REVIEW

File: scripts/india_udise/common/base_school_data.py [hunk 1]
```diff
@@ -104,6 +104,45 @@ class GeographicLevel(enum.Enum):
     BLOCK = "block"
 
 
+ATTRIBUTE_MAPPING = {
+    "cat1": {
+        "levelOfSchool": "PrimarySchool_Grade1To5"
```
Comment (ajaits) [inline scripts/india_udise/common/base_school_data.py:109]: Would the prefix 'UDISE_' be added later?
The enums are: https://datacommons.org/browser/UDISE_PrimarySchool_Grade1To5

File: scripts/india_udise/common/base_school_data.py [hunk 2]
```diff
@@ -104,6 +104,45 @@ class GeographicLevel(enum.Enum):
...
+    },
+    "cat6": {
+        "levelOfSchool": "SeniorSecondarySchool_Grade1To10"
+    },
+    "cat7": {
+        "levelOfSchool": "SeniorSecondarySchool_Grade6To10"
+    },
+    "cat8": {
+        "levelOfSchool": "SeniorSecondarySchool_Grade9To10"
+    },
```
Comment (ajaits) [inline scripts/india_udise/common/base_school_data.py:131]: if 'cat9' is expected to be dropped, pls add a  comment.

**Review by ajaits — APPROVED** [review]

File: scripts/india_udise/common/base_school_data.py (hunk 1 above)
Comment (thejeshgn) [reply scripts/india_udise/common/base_school_data.py:109]: Yes. In [this line](https://github.com/datacommonsorg/data/blob/307fd81d31edfecf1b4fc20564cc06f2014426de/scripts/india_udise/common/base_school_data.py#L290)

File: scripts/india_udise/common/base_school_data.py (hunk 2 above)
Comment (thejeshgn) [reply scripts/india_udise/common/base_school_data.py:133]: Done. 

udise_school_category

**Comment by thejeshgn** [comment]
@ajaits and @pradh please ignore this. Dont merge. The rebase has all the old ones too. I will do a new MR with just my changes.

**Comment by thejeshgn** [comment]
Here is the new MR - https://github.com/datacommonsorg/data/pull/563

**Pull request closed without merging**